recipes-images/images/torizon-minimal.bb

```diff
@@ -3,6 +3,6 @@ DESCRIPTION = "Torizon OS without a container engine. \
 All other Torizon OS features are included, such as remote and offline OS updates, \
 device monitoring, and more."
 
-require torizon-core-common.inc
+require torizon.inc
```

Comment:
We already have a `torizon.inc` for distro configuration (under `conf/distro/include`) so I'd suggest using a different name here to avoid confusion (e.g. `torizon-image.inc` since this would be a include for image recipes).

Reply:
I've renamed it to `torizon-base.inc`.